life/updated/02-life.adoc

```diff
@@ -1568,447 +1566,444 @@ for we were all committed to prison.
 or taken at an unlawful assembly, with force of arms,
 in contempt of the king and his laws, crown and dignity,
-to the terror of the people,`" etc.
-And he said to the jury, "`Gentlemen, you have heard their indictment,
-if you find them guilty, you find for the king.`"
+to the terror of the people, etc.
+And he said to the jury, "`Gentlemen, you have heard their indictment.
+If you find them guilty, you find for the king.`"
 And a bishop who sat upon the bench with the judge, stood up and said,
-"`That the first Quaker that ever was in England,
+"`The first Quaker that ever was in England
 was hanged for being concerned in the Popish plot.`"
-I answered, that the first who was called a Quaker, was now alive.
+I answered that the first who was called a Quaker was still alive.
 He said again,
-"`He could prove by sufficient witness that he
+"`I can prove by sufficient witness that he
 was hanged for being one in the Popish plot.`"
 Then the bishop being enraged, because he was contradicted, held up his hand towards us,
-and bid us "`Have a care what we said, for those who had estates among us,
-it should cost them their estates,
-and they that had not should lie in prison until they perished.`"
-Such was their rage and wickedness against us, that it was very grievous to hear them;
-but there was a secret cry many times ran through my heart unto the Lord, "`Lord,
-work for your name`'s sake, and confound their wisdom, and rage,
-and bring down their proud and wicked spirits,
-and bring to naught their mischievous contrivance,
-that they have been contriving against your innocent people,
-as they have been making themselves merry, and drinking wine to the full,
+and bid us have a care what we said, for to those who had estates among us,
+it would cost them their estates,
+and they that had none would lie in prison until they perished.`"
+Such was their rage and wickedness against us, that it was very grievous to hear them,
+but this was a secret cry many times which ran through my heart unto the Lord, "`Lord,
+work for Your name`'s sake, and confound their wisdom, and rage,
+and bring down their proud and wicked spirits.
+Bring to naught their mischievous contrivance
+that they have been contriving against Your innocent people,
+as they have been making themselves merry and drinking wine to the full,
 and feeding themselves with the fatness of the earth, as the rich man did,
-and have what their hearts lust after,
-and yet none of all these things will give them content nor satisfaction,
+and have what their hearts lust after.
+And yet none of all these things will give them content nor satisfaction
 but the destruction of a poor despised people.
-Oh Lord! make your power known this day, and that which will make most for your honor,
-and the prosperity of your blessed truth, do you bring to pass;
+Oh Lord, make Your power known this day, and that which will make most for Your honor,
```

Comment:
Here I would put back in the "bring to pass" phrase, but put it up a line, so that it reads like: Oh Lord, make Your people known this day, **and bring to pass that which** will make most for Your honor and the prosperity....